CustomBatchWatchdog.cs

```diff
@@ -117,6 +123,8 @@ private void LoadConfigFromFile()
         private void Recover(RecoveryItem rc)
         {
             ApplicationLoader.PROCESS_INFORMATION procInfo;
+            if (rc.overrideRecoveryExecutionTimeout != 0)
+                recoveryExecutionTimeout = rc.overrideRecoveryExecutionTimeout;
```

Comment:
Please use the [Allman style](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) when it comes to braces, you even did it between rows 76-79 in this file.

```csharp
if (rc.OverrideRecoveryExecutionTimeout != 0)
{
    recoveryExecutionTimeout = rc.OverrideRecoveryExecutionTimeout;
}
```

Reply:
Done